modules/virt-example-nmstate-IP-management.adoc

```diff
@@ -91,33 +91,156 @@ The following snippet configures an Ethernet interface that uses a dynamic IP ad
 [id="virt-example-nmstate-IP-management-dns_{context}"]
 == DNS
 
-Setting the DNS configuration is analagous to modifying the `/etc/resolv.conf` file. The following snippet sets the DNS configuration on the host.
-
+By default, the `nmstate` API stores DNS values globally as against in a network interface. For certain situations, you must configure a network interface to store DNS values. To define a DNS configuration for a network interface, you must initially specify the `dns-resolver` section in the network interface's YAML configuration file as shown in the following example:
++
 [source,yaml]
 ----
 # ...
-    interfaces: <1>
-       ...
-       ipv4:
-         ...
-         auto-dns: false
-         ...
-    dns-resolver:
-      config:
-        search:
-        - example.com
-        - example.org
-        server:
-        - 8.8.8.8
+dns-resolver:
```

Comment:
I placed this section here to avoid repitition in examples. If you see any other segments I could remove, please let me know as reducing the YAML sizes would make for a better scanning experience of the docs.

Reply:
I like where you placed this example. I agree that reducing the YAML sizes makes for a better scanning experience, though I can't find other segments worth removing, as they all look like helpful examples. (And I know that at least in ROSA, the feedback we've been hearing from clients is to have more examples.)

Reply:
Thanks. This was one approach to reduce the repetition of applying this codeblock to all the examples in this section. I'll let the SME decide on this one.

@cybertron